src/test/java/com/stripe/StripeTest.java

```diff
@@ -69,6 +69,10 @@
 import java.util.List;
 import java.util.Map;
 import java.util.UUID;
+import java.util.Calendar;
```

Comment:
And if possible, mind just keeping this imports alphabetized? Otherwise, looking good. Thanks a lot for the contributions!!